Unable to release due to JavaDoc errors

We are currently unable to release DAMAP due to [failures in the release pipeline](https://github.com/damap-org/damap-backend/actions/runs/15589248278/job/43903674622). These failures stem from the need to generate JavaDoc, which seems to conflict with the code generator:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.7.0:jar (attach-javadocs) on project base: MavenReportException: Error while generating Javadoc: 
Error:  Exit code: 1
Error:  /home/runner/work/damap-backend/damap-backend/target/generated-sources/jsonschema2pojo/org/damap/base/rest/madmp/dto/Cost.java:20: warning: empty <p> tag
Error:   * <p>
Error:     ^
Error:  /home/runner/work/damap-backend/damap-backend/target/generated-sources/jsonschema2pojo/org/damap/base/rest/madmp/dto/Project.java:20: warning: empty <p> tag
Error:   * <p>
Error:     ^
Error:  /home/runner/work/damap-backend/damap-backend/target/generated-sources/jsonschema2pojo/org/damap/base/rest/madmp/dto/TechnicalResource.java:18: warning: empty <p> tag
Error:   * <p>
Error:     ^
Error:  /home/runner/work/damap-backend/damap-backend/target/generated-sources/jsonschema2pojo/org/damap/base/rest/madmp/dto/SecurityAndPrivacy.java:18: warning: empty <p> tag
Error:   * <p>
Error:     ^
Error:  /home/runner/work/damap-backend/damap-backend/target/generated-sources/jsonschema2pojo/org/damap/base/rest/madmp/dto/Contact.java:18: warning: empty <p> tag
Error:   * <p>
Error:     ^
```

--- a/pom.xml
+++ b/pom.xml
@@ -221,6 +221,20 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <version>3.7.0</version>
+        <executions>
+          <execution>
+            <id>attach-javadocs-verify</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
         <groupId>${quarkus.platform.group-id}</groupId>
         <artifactId>quarkus-maven-plugin</artifactId>
         <version>${quarkus.platform.version}</version>

--- a/src/main/java/org/damap/base/conversion/AbstractTemplateExportSetup.java
+++ b/src/main/java/org/damap/base/conversion/AbstractTemplateExportSetup.java
@@ -95,7 +95,7 @@ public abstract class AbstractTemplateExportSetup extends AbstractTemplateExport
    * getContributorsByRole.
    *
    * @param contributors a {@link java.util.List} object
-   * @param role a {@link org.damap.base.enums.EContributorRole} object
+   * @param roles a {@link java.util.Set} object
    * @return a {@link java.util.List} object
    */
   protected List<Contributor> getContributorsByRole(

--- a/src/main/resources/org/damap/base/spec/oaf/oaf-common-1.0.xsd
+++ b/src/main/resources/org/damap/base/spec/oaf/oaf-common-1.0.xsd
@@ -221,8 +221,7 @@
 
     <xs:complexType name="relToType">
         <xs:annotation>
-            <xs:documentation>Information about the related entity. <p>The semantics of the
-                relationships is expressed by the attributes class and scheme. </p>
+            <xs:documentation>Information about the related entity. <p>The semantics of the relationships is expressed by the attributes class and scheme. </p>
             </xs:documentation>
         </xs:annotation>
         <xs:simpleContent>

